doc/installing/server.md

```diff
@@ -86,14 +86,35 @@ Mapping of unix groups isn't supported at the moment.
 
 To add a web path prefix to the usual webui web path set ``WEBUI_PATHPREFIX`` variable into ``openquake/server/local_settings.py`` to a prefix path starting with ``/`` and ending without it (e.g. ``'/path/prefix'``); the same variable should be set as environment variable.
 
+#### Configure the directory to store the server user access log
+
+By default, user access information is logged through the standard Django logger. In oder to write such information to a file, for instance to be digested by Fail2Ban, the variable `WEBUI_ACCESS_LOG_DIR` must be specified in `local_settings.py`, e.g.:
+```python
+WEBUI_ACCESS_LOG_DIR = '/var/log/oq-engine'
+```
+In that case the file `webui-access.log` will be created inside the specified directory.
+Please note that the directory must be created if it does not exist yet, e.g.:
+```bash
+$ sudo mkdir /var/log/oq-engine
+```
+Furthermore, the user `openquake` must own that directory.
 
 ## Running in production
 
 On a production system [nginx](http://nginx.org/en/) + [gunicorn](http://gunicorn.org/) is the recommended software stack to run the WebUI.
 
 #### gunicorn
 
-*gunicorn* can be installed either via `pip` or via the system packager (`apt`, `yum`, ...). When using `python-oq-libs` for RedHat or Debian *gunicorn* is already provided.
+*gunicorn* can be installed either via `pip` or via the system packager (`apt`, `yum`, ...). For example:
+
+```bash
+$ sudo su -
+# source /opt/openquake/venv/bin/activate
+# pip install gunicorn
+# deactivate
```

Comment:
Why the prompt changes from `$` to `#` ? Looks like an error.

Reply:
The syntax highlighting here makes those lines look like comments. However, it is quite standard that the prompt for a standard user is `$` whereas the prompt for root is `#`. So in principle it looks correct to me.

Reply:
Using 'console' for the syntax highlighting, it looks much better